docs/guides/connecting_to_your_data/database/postgres.md

```diff
@@ -1,152 +1,111 @@
 ---
 title: How to connect to your data in a postgresql database
 ---
-import AddingCredentials from '../reference/database_credentials.md'
-import NextSteps from '../reference/link-to-validator.md'
-import ConnectionStringDetails from '../reference/configuration_explanation.md'
+import NextSteps from '../components/next_steps.md'
+import Congratulations from '../components/congratulations.md'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This guide will help you connect Great Expectations to data in a Postgresql database. You will be building a `Datasource` configuration, and using it to verify connectivity. By the end of this how-to guide, you should have a `Batch` representing a slice of data in your database, and a working code snippet that can be used as a starting point for more complex configurations.
+This guide will help you connect to data in a Postgresql database.
+This will allow you to validate data in tables or from queries.
 
-:::note Prerequisites: This how-to-guide assumes you have already:
+:::note Prerequisites: This guide assumes you have already:
 - Completed the [Getting Started Tutorial.](../../tutorials/quick-start.md)
 - Have a working installation of Great Expectations.
-- Have data on Postgresql database that you would like to connect to.
+- Have access to data in a Postgres database.
 :::
 
 ## Steps
 
-### 1. Install required packages
+### 1. Install required dependencies
 
-Next make sure you have the necessary packages for Great Expectations to connect to your postgres database.
+First, install the necessary dependencies for Great Expectations to connect to your postgres database.
 
 ```console
 pip install sqlalchemy psycopg2
 ```
 
-### 2. Determine how to add credentials to configuration
+### 2. Choose a credential handling method
```

Comment:
A bit nit-picky, but this title sounds a bit .. clunky for me.

Reply:
Agreed!